Automattic/wp-calypso
# FSE: Introduce a new block Posts Carousel

#### Changes proposed in this Pull Request

* First pass at importing Newspack's Carousel block.

#### Testing instructions



* Make sure you have a post or two with a featured image
* Run `$ yarn && yarn run sync:newspack-blocks --nodemodules` to sync the blocks.
* Run `$ yarn dev` to build the plugin. 
* See if blog posts block and posts carousel block appear in the template selector.

Fixes #40979.

Patch: D42556-code
